src/tests/test_dte_data_models.py

```diff
@@ -1617,24 +1763,26 @@ def test_validate_referencias_rut_otro_is_consistent_with_emisor_rut(self) -> No
 
         expected_validation_errors = [
             {
-                'loc': ('__root__',),
-                'msg': "'rut_otro' must be different from 'emisor_rut':"
-                " Rut('60910000-1') == Rut('60910000-1'),"
-                " 'Referencia' number 1.",
+                'loc': (),
+                'msg': "Value error, 'rut_otro' must be different from 'emisor_rut': "
+                "Rut('60910000-1') == Rut('60910000-1'), 'Referencia' number 1.",
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 referencias=[obj_referencia],
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_url=False,
+            include_context=False,
+            include_input=False,
```

Comment:
Sort lexicographically.

```suggestion
            include_context=False,
            include_input=False,
            include_url=False,
```